### DISCUSSION TITLE
1.6.0 Release

### DISCUSSION THREAD

**Discussion by igor-lemon** [DockStation/dockstation#310] [Show and tell]
Hi there! Here we can discuss bugs at 1.6.0 release

You can get RC version here.
https://github.com/igor-lemon/pre-releases/releases/

The milestone
https://github.com/DockStation/dockstation/milestone/5

**Comment by softexpert** [comment]
Thank you for your efforts, man !

I just installed the  1.6.0-rc.0 over 1.5.1. 
First start is quite disconcerting - nothing much happens on the interface.
Restart brings up the interface with the error message about Compose 2 not working and the button to enable Compose version 1 is clearly visible and effective.
Starting from there, it seems to function as the previus version, but:
- a click on the info button will crash something and the whole interface remains blanc
![image](https://user-images.githubusercontent.com/1254018/162559050-65d044f3-4393-4402-b402-f142c48caab0.png)

- once the interface is blanc, nothing will revive it (the top menus have no effect), so the application must be restarted

Again, thanks for the efforts and really sorry that you cannot afford to continue this project !

**Reply by igor-lemon** [reply]
> First start is quite disconcerting - nothing much happens on the interface.

I didn’t quite understand what you mean, was there a white screen or infinity loader or Compose Error?

> a click on the info button will crash something and the whole interface remains blanc

Ok, I'll check.

>once the interface is blanc, nothing will revive it (the top menus have no effect), so the application must be restarted

Yep if throws Unhandled Exception the app is down

>Again, thanks for the efforts and really sorry that you cannot afford to continue this project !

Don't bury the project ahead of time :)

**Reply by softexpert** [reply]
> > First start is quite disconcerting - nothing much happens on the interface.
> 
> I didn’t quite understand what you mean, was there a white screen or infinity loader or Compose Error?
> 
The interface was looking a bit like this 
![image](https://user-images.githubusercontent.com/1254018/162568013-ccd43f49-5fef-4c76-9d99-12720042c284.png)
and nothing seemed to happen.
If there is a flag that enables more verbose traces, I could try to collect more meaningful information for you ...

Fo now, if I start from command line I get the following
```bash
(node:307623) electron: The default of contextIsolation is deprecated and will be changing from false to true in a future release of Electron.  See https://github.com/electron/electron/issues/23506 for more information
Failed to fetch extension, trying 4 more times
Failed to fetch extension, trying 4 more times
Failed to fetch extension, trying 3 more times
Failed to fetch extension, trying 3 more times
Failed to fetch extension, trying 2 more times
Failed to fetch extension, trying 2 more times
Failed to fetch extension, trying 1 more times
Failed to fetch extension, trying 1 more times
Failed to fetch extension, trying 0 more times
An error occurred:  Error: net::ERR_BLOCKED_BY_RESPONSE
    at SimpleURLLoaderWrapper.<anonymous> (electron/js2c/browser_init.js:105:6536)
    at SimpleURLLoaderWrapper.emit (events.js:315:20)
undefined
Failed to fetch extension, trying 0 more times
An error occurred:  Error: net::ERR_BLOCKED_BY_RESPONSE
    at SimpleURLLoaderWrapper.<anonymous> (electron/js2c/browser_init.js:105:6536)
    at SimpleURLLoaderWrapper.emit (events.js:315:20)
```

> > a click on the info button will crash something and the whole interface remains blanc
> 
> Ok, I'll check.
> 
> > once the interface is blanc, nothing will revive it (the top menus have no effect), so the application must be restarted
> 
> Yep if throws Unhandled Exception the app is down
> 
> > Again, thanks for the efforts and really sorry that you cannot afford to continue this project !
> 
> Don't bury the project ahead of time :)

Far from me the thought ! :)

**Comment by igor-lemon** [comment]
+1 bug
Doesn't save YAML content at Editor

**Comment by igor-lemon** [comment]
RC.1 Is ready

https://github.com/igor-lemon/pre-releases/releases/tag/v1.6.0-rc.1

Fixes:
1) Reported bugs from @SoftExpert 
2) Improved Docker V2 availability check and applying it
3) Fixed clearing an overridden compose config
4) Fixed content saving at Editor
5) Removed project name applying (-p flag) for better compatibility with CLI Docker Compose
6) Change "not-allowed" to default cursor above disabled fields at Config tab

**Comment by igor-lemon** [comment]
RC.2 is ready 

https://github.com/igor-lemon/pre-releases/releases/tag/v1.6.0-rc.2

Fixes:
1) Fix scroll on the project settings page
2) Fix Ports page
3) Add "Add Connection" button on the Containers page

**Comment by docertabum** [comment]
Hi @igor-lemon ,
so far so good. I didnt have much time during the weekend to play with it, but at least for me entering CLI(exec) on a container doesn't work. This is what I see every time:
<img width="378" alt="image" src="https://user-images.githubusercontent.com/588442/162684173-6ca26d18-c253-4706-980c-43070a5f46c3.png">

bug replicable in rc0 & rc2

Wish list:
- invoking search in logs (command+f) button
- when searching through logs I would prefer string highlighting instead of log filtering :)

**Reply by igor-lemon** [reply]
@docertabum 
>so far so good. I didnt have much time during the weekend to play with it, but at least for me entering CLI(exec) on a container doesn't work. This is what I see every time:

+++

> invoking search in logs (command+f) button

+++

> when searching through logs I would prefer string highlighting instead of log filtering

I agree, it's relevant case. Need to think how to implement it in UI

**Reply by igor-lemon** [reply]
Up. Sorry, about "Exec failed" error, didn't notice "ENOENT". Yep, looks like I lose a script.

**Comment by igor-lemon** [comment]
RC.3 is ready

https://github.com/igor-lemon/pre-releases/releases/tag/v1.6.0-rc.3

1) Added migration from 1.5.x to 1.6.x (Thanks @malko)
2) Fixed exec container (Thanks @docertabum)
3) Added CTRL/CMD + F hotkey to toggle search logs bar (Thanks @docertabum)
4) Fixed logo on the initializing page

**Comment by igor-lemon** [comment]
Hi guys! Have you caught any bugs yet?

**Comment by malko** [comment]
@igor-lemon  For me Docker compose error at launch time as still using version 1 but let me choose and then relaunch.
After that it didn't detect running projects, but changing to container tab, and all active containers are listed as active. Switching back to projects show running projects correctly. Visualisation of project infra is broken (no viz at all) , and clicking on a project is slow compared to previous version. Stats monitor seems ok. 
Trying to edit docker-compose.yml load an empty document with only the version in it.
Hope this help

**Reply by igor-lemon** [reply]
It's strange because I had a few same troubles and it was fixed...
Did you use the latest RC version?

**Comment by docertabum** [comment]
Hello Igor,
I am just experimenting with Colima on my Mac. Would it be possible to use it with DockStation?
<img width="754" alt="image" src="https://user-images.githubusercontent.com/588442/166886364-fad6682e-de6d-42c1-b7cb-d4024510210e.png">
<img width="1012" alt="image" src="https://user-images.githubusercontent.com/588442/166886888-1f7b6eed-b52d-4a7d-b58b-477b534600b7.png">

**Reply by igor-lemon** [reply]
Hi, check https://github.com/DockStation/dockstation/issues/18 please
